lib/ansible/plugins/callback/logstash.py

```diff
@@ -21,19 +22,27 @@
         env:
           - name: LOGSTASH_SERVER
         default: localhost
+        ini:
+          - section: callback_logstash
+            key: server
```

Comment:
missing `version_added:`

Reply:
Added version 2.10 because has the label affects_2.10. Hope it's right.